### ISSUE TITLE
浏览器不支持document.registerElement方法了

### ISSUE THREAD

**Issue by BruceGuanFF** [alibaba/JS-Memory-Analysor#1]
浏览器不支持document.registerElement方法了

**Comment by callum-atwal** [comment]
If you just need to quickly use this without fixing this bug, use an older version of Chrome/Chromium:
eg:
https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Mac/690231/

**Comment by zhenjia001** [comment]
older version Chrome is ok?

**Comment by callum-atwal** [comment]
Yeah, click that link above and download an old version of chromium
